packages/grid/src/index.ts

```diff
@@ -2,10 +2,7 @@ import Grid from './Grid';
 import { gridThemeNamespace } from './theme';
 
 
-export { GridSort } from './consts';
-
 export * from './interfaces';
-export {
-  gridThemeNamespace,
-};
+export { GridSort, GridPositions } from './consts';
+export { gridThemeNamespace };
```

Comment:
MB change 
`export { gridThemeNamespace };`
to
`export { gridThemeNamespace }  from './theme';`

and remove import the above?